lib/cc/config/default.rb

```diff
@@ -1,57 +0,0 @@
-module CC
-  class Config
-    class Default < Config
-      DUPLICATION_LANGUAGES = %w[
```

Comment:
Mentioned in a commit message, but we don't need these anymore: `duplication` provides a default itself if one is not considered.